package.json

```diff
@@ -53,12 +53,13 @@
     "version:patch": "npm version patch",
     "version:minor": "npm version minor",
     "xpostversion": "npm publish && npm run deploy",
-    "predeploy": "npm run build && copyfiles -f node_modules/three/build/three.module.js demo/build/three/build && copyfiles -f dist/gcode-preview.es.js demo/build && copyfiles -f node_modules/canvas2image/canvas2image.js demo/build/canvas2image",
+    "predeploy": "npm run build && copyfiles -f node_modules/three/build/three.module.js demo/build/three/build && copyfiles -f dist/gcode-preview.es.js demo/build && copyfiles -f node_modules/canvas2image/canvas2image.js demo/build/canvas2image && copyfiles -f node_modules/lil-gui/dist/lil-gui.esm.min.js demo/build/lil-gui/dist",
     "deploy": "firebase deploy --only hosting",
     "typedoc": "typedoc",
     "typedoc:watch": "typedoc --watch"
   },
   "dependencies": {
-    "three": "^0.155.0"
+    "three": "^0.159.0",
+    "lil-gui": "^0.19.2"
```

Comment:
The upgrade of three.js changed the location and version of lil-gui. Since we're using it directly, it's safer to list it as a dependency.